modules/installation-aws-arm-tested-machine-types.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// aws modules
+
+[id="installation-aws-arm-tested-machine-types_{context}"]
+= Tested instance types for AWS ARM
+
+The following Amazon Web Services (AWS) ARM instance types have been *tested* with {product-title}.
+
+.Machine types based on arm64 architecture
+[%collapsible]
+====
+include::https://raw.githubusercontent.com/openshift/installer/master/docs/user/aws/tested_instance_types_aarch64.md[]
+====
```

Comment:
@yunjiang29  the tested_instance_types_aarch64.md is family name, shall we change the md file name?